src/modules/_nomodule/PixelManipulation.js

```diff
@@ -113,25 +119,27 @@ module.exports = function PixelManipulation(image, options) {
           ).then(bytes =>
             WebAssembly.instantiate(bytes, imports)
           ).then(results => {
-            console.log('yes');
             results.instance.exports.manipulatePixel(pixels.shape[0], pixels.shape[1], inBrowser, test);
             extraOperation();
           }).catch(err => {
             perPixelManipulation();
```

Comment:
Hmm, is `catch` really a good way to detect compatibility? What if it fails for another reason? 

For now, why don't we log out a message like `WebAssembly acceleration errored; falling back to JavaScript in PixelManipulation` plus the actual error message, so that it's clear what happened.

Reply:
Here and in the other instance of `perPixelManipulation`!

Can you also comment at `perPixelManipulation()` to make it clear that it's the pure JavaScript fallback?

```suggestion
            perPixelManipulation(); // pure JavaScript fallback
```

Reply:
Sure!!